action.yml

```diff
@@ -122,11 +123,12 @@ runs:
         GITHUB_REPOSITORY: ${{ github.repository }}
       run: |
         # https://docs.github.com/en/rest/git/tags
+        # tags are sorted with most recent tags first
         RESP_CODE=$(curl -w %{http_code} -s -L -o __tags.json \
           -H "Accept: application/vnd.github+json" \
           -H "Authorization: Bearer ${{ github.token }}" \
           -H "X-GitHub-Api-Version: 2022-11-28" \
-          https://api.github.com/repos/$GITHUB_REPOSITORY/tags \
+          https://api.github.com/repos/$GITHUB_REPOSITORY/tags?per_page=100 \
```

Comment:
This isn't required, though it may prevent some other regression in the future as more tags are added